transformations/suspecting_paraphraser/README.md

```diff
@@ -0,0 +1,18 @@
+# Suspecting Paraphraser 🦎  + ⌨️ → 🐍
+This paraphraser transforms a yes/no question into a subjunctive one, which helps to add more question specific informality to the dataset. 
+
+Example: "Did the American National Shipment company really break its own fleet?" -> "The American National Shipment company really broke its own fleet, didn't it?"
+
+
+Author name: Witold Wydmański
+Author email: witold.wydmanski@uj.edu.pl
+Author Affiliation: Jagiellonian University
+
+## What type of a transformation is this?
+This transformation acts like a paraphraser and makes lexical substitutions.
+
+## What tasks does it intend to benefit?
+This transformation would benefit tasks that are focused on conversational question answering.
+
+## What are the limitations of this transformation?
+The pronouns used in the endings aren't always on point - mostly because of ambiguity. As such, this transformation shouldn't be used for tasks that require their correctness. 
```

Comment:
Are you referring to the pleonastic "it" here? Could you explain this through an example so that what you mean by "pronouns used in the endings" will be clarified better [and also the limitation would be overall explained better]

Reply:
Yes, what I meant was that while this paraphraser tries to be consistent with the pronouns, there are some cases in which it will give an unexpected result.

Examples:
 - Did he drink that tea? -> He drank that tea, didn't he? (The pronoun gets recognized correctly)
 - Has Tesla finished high school? -> Tesla has finished high school, didn't it? (because the word Tesla gets recognized as an organization entity)
 - Was Sally here yesterday? -> Sally was here yesterday, wasn't [she/he/it]? (Chosen at random, because any pronoun can match here. By default, names that are recognized as strongly masculine or feminine will get 90% chance of selecting corresponding pronoun, but it's configurable).

Do you think that I should include those examples in the readme?

Reply:
Certainly, you should, these will be really helpful! Yeah I get it better why they would now, since finding and assigning both gender and PER vs ORG would be noisy. In addition to the examples, it would be great if you could mention these underlying reasons too.

Reply:
OK, I've update readme with those examples. Thank you for the comment!